apollo-runtime/src/main/java/com/apollographql/android/ApolloClient.java

```diff
@@ -0,0 +1,95 @@
+package com.apollographql.android;
+
+import com.apollographql.android.api.graphql.Mutation;
+import com.apollographql.android.api.graphql.Operation;
+import com.apollographql.android.api.graphql.Query;
+import com.apollographql.android.api.graphql.ResponseFieldMapper;
+import com.apollographql.android.api.graphql.ScalarType;
+
+import java.lang.reflect.Type;
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+import javax.annotation.Nonnull;
+
+import okhttp3.Call;
+import okhttp3.HttpUrl;
+import okhttp3.OkHttpClient;
+
+import static com.apollographql.android.Utils.checkNotNull;
+
+/** TODO */
+public final class ApolloClient {
```

Comment:
Don't need but would be nice to have interface that defines that ApolloClient is a factory that creates ApolloCall.  In case we ever want to migrate to another Client that can also produce ApolooCall